java-2016/homework-9/Tema 9 IO/src/com/example/JunitTestClass.java

```diff
@@ -0,0 +1,62 @@
+package com.example;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.List;
+
+/**
+ * Created by Oana on 2/28/2017.
+ */
+public class JunitTestClass {
+
+
+    @Test
+    public void readNumber() throws ParseException {
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Oana" + File.separator + "Desktop" + File.separator + "Tema 9 IO" + File.separator + "TheAthlete.csv");
+        Main.Read myReader = new Main.Read();
```

Comment:
For unit tests there should be no interactions with files. You should just define a list of athletes in memory.